From e198571006adbcd8be392f2e4f7763acdd78daf9 Mon Sep 17 00:00:00 2001
From: Neil Mayhew <neil_mayhew@sil.org>
Date: Tue, 29 Mar 2011 14:29:13 -0600
Subject: [PATCH 1/1] Fix missing -lglib-2.0 in tests

Makefile.am neglects to include link FLAGS from configure.
This first showed up as a problem in wheezy.
---
 tests/Makefile.am |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 5b8c67c..7f28b18 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -11,7 +11,8 @@ DEPS =						\
 
 
 LDADDS =					\
-	$(top_builddir)/src/libgdiplus.la
+	$(top_builddir)/src/libgdiplus.la \
+	$(GDIPLUS_LIBS)
 
 noinst_PROGRAMS =			\
 	testgdi testbits testclip testreversepath
-- 
1.7.4.1

